integration_tests/cases/common/function/data_bin.result

```diff
@@ -0,0 +1,63 @@
+DROP TABLE IF EXISTS `02_function_date_bin_table`;
+
+affected_rows: 0
+
+CREATE TABLE `02_function_date_bin_table` (
+  `timestamp` timestamp NOT NULL,
+  `value` int,
+  timestamp KEY (timestamp)) ENGINE=Analytic
+WITH(
+	 enable_ttl='false'
+);
+
+affected_rows: 0
+
+INSERT INTO `02_function_date_bin_table`
+(`timestamp`, `value`)
+VALUES
+    (1659577423000, 1),
+    (1659577422000, 2),
+    (1659577320000, 3),
+    (1659571200000, 4),
+    (1659484800000, 5),
+    (1656777600000, 6);
+
+affected_rows: 6
+
+SELECT DATE_BIN(INTERVAL '30' second, `timestamp`, TIMESTAMP '2001-01-01T00:00:00Z') as `time` FROM `02_function_date_bin_table`;
+
+time,
+Timestamp(1656777600000),
```

Comment:
Could you rewrite all those query as
```sql
SELECT 
timestamp,
DATE_BIN(INTERVAL '30' second, `timestamp`, TIMESTAMP '2001-01-01T00:00:00Z') as `time`
FROM `02_function_date_bin_table`
order by timestamp;
```
Your current test case have no specific order, and is hard to tell whether it's right.